cibuildwheel/linux.py

```diff
@@ -103,10 +103,10 @@ def build(project_dir, package_name, output_dir, test_command, test_requires, be
             pybin_paths=' '.join(c.path+'/bin' for c in platform_configs),
             test_requires=' '.join(test_requires),
             test_command=shlex_quote(
-                prepare_command(test_command, python='python', pip='pip', project='/project') if test_command else ''
+                prepare_command(test_command, project='/project') if test_command else ''
```

Comment:
Maybe it's possible to squeeze on a single line now, if you care about such things ;-) If not, eh...